haiku-misc/neonlights/neonlights-0.0.1.recipe

```diff
@@ -0,0 +1,42 @@
+SUMMARY="Screensaver drawing neon lines"
+DESCRIPTION="NeonLights is a screensaver that draws colorful neon lines. \
+It was based loosely on Jared Tarbell's work. Inspired by Atalax's Substrate."
+HOMEPAGE="https://github.com/pulkomandy/neonlights"
+COPYRIGHT="2017 Adrien Destugues"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="f658b45894fe5d2349be4e06c3a0bce359985e27"
+SOURCE_URI="https://github.com/pulkomandy/neonlights/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="9271f3778218bb63430da00d00b099c5d3e9d83b21d6c132533069b237199663"
+SOURCE_DIR="neonlights-$srcGitRev"
```

Comment:
@dash102, you can switch to version 0.2 which @pulkomandy has just released ;-)
For that purpose, you just need to:

1) drop the line which defines srcGitRev.
2) change SOURCE_URI and CHECKSUM_SHA256 to:
```
SOURCE_URI="https://github.com/pulkomandy/neonlights/archive/v$portVersion.tar.gz"
CHECKSUM_SHA256="a7a8f7faab4ff56d8677fc64cbb1759c7532f8db7309a467199b0c9b1cac8547"
```
3) drop the line which defines SOURCE_DIR, because the default value, neonlights-0.2, will match what is in the archive.
4) insert after CHECKSUM_SHA256, that is, right where you had the SOURCE_DIR, this line:
```
SOURCE_FILENAME="neonlights-$portVersion.tar.gz"
```

Steps 1 to 3 are required. Step 4 is not required but is recommended. It tells haikuporter that once it has downloaded the "v0.2.tar.gz" tarball it should rename it to "neonlights-0.2.tar.gz".

Of course, there is a last step: you need rename the recipe to neonlights-0.2.recipe, but you have to use **git mv** for that:
```
cd haikuports/haiku-misc/neonlights
git mv neonlights-0.0.1.recipe neonlights-0.2.recipe
git commit --amend --no-edit neonlights-0.0.1.recipe neonlights-0.2.recipe
```
Sorry if all this was obvious. I'm guiding you in the hope that you will enjoy updating this recipe, so that you will enjoy making even more contributions in the future ;-)

BTW, everything else is OK, so I guess your PR will be merged soon after you make the changes detailed in this message ;-)